models/repo.go

```diff
@@ -166,6 +172,9 @@ type Repository struct {
 	CloseIssuesViaCommitInAnyBranch bool               `xorm:"NOT NULL DEFAULT false"`
 	Topics                          []string           `xorm:"TEXT JSON"`
 
+	// Avatar
+	Avatar string `xorm:"VARCHAR(2048)"`
```

Comment:
Does the full URL need to be stored in DB? For user avatars this makes sense due to gravatar (and other external avatar services), but for repo avatars it follows the following patter `setting.AppSubURL + "/repo-avatars/" + repo.Avatar`

Reply:
You mean that it easily fits into 250 symbols?

Reply:
Yes, rather than storing the full URL, just the filename could be stored.

Reply:
Done in https://github.com/go-gitea/gitea/pull/6986/commits/4ae938309774a9d2e325fbbfcc36278294d728e6